src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -8090,6 +8086,12 @@ void ExportMusicXml::writeMeasureTracks(const Measure* const m,
                 continue;
             }
 
+            EngravingItem* ic = seg->findAnnotation(ElementType::INSTRUMENT_CHANGE, strack, etrack - 1);
```

Comment:
I think these additions should be moved below the call to `moveToTickIfNeed`

Reply:
agreed